GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -581,6 +584,18 @@ private void FileTreeContextMenu_Opening(object sender, System.ComponentModel.Ca
             openWithDifftoolToolStripMenuItem.Visible = isFile;
             openWithToolStripMenuItem.Visible = isFile;
             openWithToolStripMenuItem.Enabled = isExistingFileOrDirectory;
+            var fsi = _rememberFileContextMenuController.getFileStatusItem(gitItem.Name, _revision);
```

Comment:
Else does not work for files from subfolders. More places?
```suggestion
            var fsi = _rememberFileContextMenuController.getFileStatusItem(gitItem.FileName, _revision);
```

Reply:
Name and FileName is assigned the same name when creating the GitItem, then the FileName is adjusted, so I missed that...

DiffTool has the same problem (another snippet I looked at). Will make a separate commit for that

Reply:
Does not work yet because path names from the FileTree contain backslashes, which git does not understand. Nothing happens (but the exit code 128) e.g. with
`git difftool --gui --no-prompt -M -C "9fbdaff9940063109e0824380dec3279973656b6:GitUI\CommandsDialogs\RevisionFileTreeControl.cs" "1f135d7c04783b1446ce0c52b29138015650ac2b:GitUI/MessageBoxes.cs"`

Reply:
updated